src/routes/docs/contributing/translating/+page.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Translating Files
+---
+
+# Translating
+
+Files uses Crowding for translation, which is cloud based SaaS supports globalization of any application on any platform.
+
+## Requirements
+- Any browser that is the latest version.
```

Comment:
We usually add a space underneath headers so let's add one here for consistency.

Reply:
Done.